components/Forms/CreateMemoryForm.js

```diff
@@ -0,0 +1,67 @@
+import styled from "styled-components";
+import { StyledButtonFrame } from "../GlobalStyles";
+import MagicInput from "./MagicInput";
+
+export default function CreateMemoryForm({ onAddCreateCard }) {
+  return (
+    <StyledForm onSubmit={onAddCreateCard}>
+      <StyledFieldset>
+        <MagicInput type="checkbox" name="favorite" />
+        <MagicInput type="date" name="date" required />
+        <MagicInput placeholder="Title" type="text" name="headline" required />
+        <MagicInput
+          type="textarea"
+          name="details"
+          placeholder="...tell me more"
+          rows={"10"}
+        />
+        <StyledButtonwrapper>
+          <StyledButtonFrame type="submit">OK</StyledButtonFrame>
+          <StyledButtonFrame type="reset">RESET</StyledButtonFrame>
+        </StyledButtonwrapper>
+      </StyledFieldset>
+    </StyledForm>
+  );
+}
+
+const StyledForm = styled.form`
+  border: none;
+  outline: none;
+  width: 50vw;
+`;
+
+const StyledButtonwrapper = styled.div`
+  display: flex;
+  justify-content: space-evenly;
+  padding: 0.5em;
+`;
+const StyledFieldset = styled.fieldset`
+  display: flex;
+  flex-direction: column;
+  gap: 1.5em;
+  border: 1px solid var(--beaver3);
+  color: var(--beaver);
+
+  input {
```

Comment:
all the inputs are in MagicInput, I guess this and the following is unnecessary here now?

Reply:
Another good point. Question:

I wanted to keep the MagicInput as simple as possible with very little styling. 

In this Case MagicInput is called in the CreateForm. I have also called it in the Edit Form, but with a bit different styling.

Is that adequate or illegal? Happy to hear your thoughts on this topic

Reply:
Its late now, but yes, its perfectly fine like this .